data/tools/unit_tree/html_output.py

```diff
@@ -150,7 +150,7 @@ def website_footer():
 
 def build_timestamp():
     """Returns an element containing the current date and time."""
-    return '<div id="lastmod">Last updated on %s.</div>' % time.ctime()
+    return '<div id="lastmod">Last updated on %s.</div>' % time.ctime() # TODO?: Localize timestamp
```

Comment:
Whether UTC or the user's local time, it really ought to be specified. Plenty of times I've assumed some US time-zone (because the Anglophonic web seems to stuck with US-centric conventions more often than not), but it's actually been UTC - and vice-versa too.

https://docs.python.org/3/library/datetime.html#datetime.datetime.ctime
It appears `ctime()` explicitly states that it won't do this (specify the time-zone). From what I can see, the string is just used at the bottom of a page ('fine print') so I think `isoformat(sep=' ', timespec='seconds')` will be an improvement over `ctime()`.